pkg/migration/controllers/migrationschedule.go

```diff
@@ -174,6 +179,11 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 			suspend := true
 			remoteMigrSchedObj.Spec.Suspend = &suspend
 			remoteMigrSchedObj.Status = stork_api.MigrationScheduleStatus{}
+			objectHash, err := getMigrationScheduleHash(migrationSchedule)
```

Comment:
Do we need to add this objectHash even when there exists a MigrationSchedule on the remote side (Think of an upgrade path)?

Reply:
This will be taken care of by the reconcileRemoteMigrationSchedule. As the annotation will not be present, there will be one update happening on existing remote static copies.